koku/masu/processor/orchestrator.py

```diff
@@ -272,6 +272,21 @@ def start_manifest_processing(  # noqa: C901
             LOG.info(log_json(tracing_id, msg="found manifests", context=manifest, schema=schema_name))
 
             last_report_index = len(report_files) - 1
+            # Override the queue if we determine a provider is large based on report count
+            report_context = manifest.copy()
+            if len(report_files) > settings.XL_REPORT_COUNT:
+                LOG.info(
+                    log_json(
+                        tracing_id,
+                        msg="marking provider large as report files exceed threshold.",
+                        file_count=len(report_files),
+                        Threshold={settings.XL_REPORT_COUNT},
+                        schema=schema_name,
+                    )
+                )
+                report_context["large_provider"] = True
+                SUMMARY_QUEUE = get_customer_queue(schema_name, SummaryQueue, report_context["large_provider"])
+                REPORT_QUEUE = get_customer_queue(schema_name, DownloadQueue, report_context["large_provider"])
```

Comment:
We ought to just pass in the bool instead of doing a dictionary lookup.
```suggestion
                SUMMARY_QUEUE = get_customer_queue(schema_name, SummaryQueue, True)
                REPORT_QUEUE = get_customer_queue(schema_name, DownloadQueue, True)
```